front/src/applications/operationalStudies/views/Scenario.tsx

```diff
@@ -129,6 +135,15 @@ export default function Scenario() {
       { skip: !timetableId }
     );
 
+  useEffect(() => {
+    if (!projectId || !studyId || !scenarioId) navigate('/operational-studies/projects');
+    // redirect if projectId or studyId is not a number
```

Comment:
Move this comment above the useEffet and remove the lines 148/149
```suggestion
    // redirect if projectId, studyId or scenarioId is not a number
```